UKHomeOffice/cerberus-service
# COP-10459 : Add tests to check arrival date time on task list and details page

## Description
Add tests to check arrival date time on task list and details page

Add changes from release 1.6.7.1 to make the test code latest

## To Test
npm run cypress:runner 

run the following tests

`Should check AirPax task information with arrival details in the feature on task overview`
`Should check AirPax task information with arrival details in the past on task overview`
`Should verify Task summary of an AirPax task on task details page`


## Developer Checklist

\* Required
